code/game/objects/items/weapons/grenades/syndieminibomb.dm

```diff
@@ -24,3 +24,10 @@
 		to_chat(user, "<span class='userdanger'>As you activate the bomb, it emits a substance that sticks to your hand! It won't come off!</span>")
 		to_chat(user, "<span class='sans'>Uh oh.</span>")
 	. = ..()
+
+/obj/item/grenade/syndieminibomb/pen
+	name = "pen"
+	desc = "It's a normal black ink pen."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "pen"
+	item_state = "pen"
```

Comment:
I recommend adding more functionality to this. This is extremely inconspicuous, so someone will be confused when they just randomly blow up. Perhaps adding a longer delay, beeping noises, and a custom priming message will go a long way to improving it.

Reply:
That or do what the trick minibomb does, put it in a clearly labled box, so the original traitor knows what it is, but can be used to trick others, and not be really clear when they throw it at someone with the beeping

Reply:
I'll take those in mind, I'll make something good out of this